src/tests/p11test/p11test_case_ec_derive.c

```diff
@@ -244,75 +251,119 @@ int test_derive(test_cert_t *o, token_info_t *info, test_mech_t *mech)
 		debug_print(" [ KEY %s ] Skip key of unknown size", o->id_str);
 		return 1;
 	}
+#endif
 
-	/* Generate the peer private key */
-	if ((key = EC_KEY_new_by_curve_name(nid)) == NULL ||
-			EC_KEY_generate_key(key) != 1) {
-		debug_print(" [ KEY %s ] Failed to generate peer private key", o->id_str);
-		EC_KEY_free(key);
+/* Generate the peer private key */
```

Comment:
missing indent. Also in the next comment.

Reply:
Fixed